libs/platform/src/lib/form/multi-combobox/commons/base-multi-combobox.ts

```diff
@@ -461,7 +486,11 @@ export abstract class BaseMultiCombobox extends CollectionBaseInput implements A
 
         const map = new Map();
         map.set('query', text);
-        map.set('limit', 12);
+
+        if (!this.limitless) {
+            map.set('limit', MAP_LIMIT);
```

Comment:
It would be better if MAP_LIMIT became configurable by user via DI Token

Reply:
done

Reply:
Can you please also add one small example or description of that feature to example?